src/types/application.types.ts

```diff
@@ -0,0 +1,6 @@
+export interface ListedApplication {
```

Comment:
Any specific reason to call it `ListedApplication` and not simply `Application` or `DatasetsApplication`?

Reply:
I actually named it Application, but renamed it to match backend. We will anyway have also Application with more details

Reply:
But I am open for ideas :D

Reply:
I named it listed application as it's an abstraction of application, it contains only a few selected attributes. There will another call to fetch the complete application.

I aggree it should match the backend anyway.